app/html/select.html

```diff
@@ -53,5 +56,6 @@ <h1 class="section-header">Verify</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
     <!-- Include all compiled plugins (below), or include individual files as needed -->
     <script src="js/bootstrap.min.js"></script>
+    <script src="js/select.js"></script>
```

Comment:
Not needed to have js/bootstrap.min.js.

Reply:
The bootstrap was added because the drap and drop version of the nav bar is not working when scaled to a mobile resolution.